src/Syntax_Client/Oni_Syntax_Client.re

```diff
@@ -0,0 +1,114 @@
+/*
+ Syntax client
+ */
+
+module Core = Oni_Core;
+
+open Oni_Syntax;
+
+module LogClient = (val Core.Log.withNamespace("Syntax.client"));
+module LogServer = (val Core.Log.withNamespace("Syntax.server"));
```

Comment:
I've so far prefixed every namespace with `Oni2.`, except for the revery log which is `Revery` right now. The idea is that the first part should identify the application or library it belongs to. That's really the only convention I've used so far.

I would also call the log modules `ClientLog` and `ServerLog`, so they're more noun-like than verb-like.

Not set on any of this of course, but that's what I've done elsewhere so far.

Reply:
Sounds good! I'll stick with the convention you've started 👍